Source/Private/ASDisplayNodeInternal.h

```diff
@@ -204,9 +204,9 @@ FOUNDATION_EXPORT NSString * const ASRenderingEngineDidDisplayNodesScheduledBefo
   NSMutableArray<ASDisplayNode *> *_yogaChildren;
 #endif
 #if YOGA_TREE_CONTIGUOUS
-  YGNodeRef _yogaNode;
   __weak ASDisplayNode *_yogaParent;
   ASLayout *_yogaCalculatedLayout;
+  BOOL _yogaLayoutInProgress;
```

Comment:
Reasonable. Should we thread-affine this property? Can yoga layouts proceed in parallel?